Source/utils/soundsample.cpp

```diff
@@ -16,10 +16,48 @@
 #include "storm/storm_sdl_rw.h"
 #include "storm/storm.h"
 #include "utils/log.hpp"
+#include "utils/math.h"
 #include "utils/stubs.h"
 
 namespace devilution {
 
+namespace {
+
+constexpr float LogBase = 10.0f;
+
+// Scaling factor for attenuating volume.
+// Picked so that a volume change of -10 dB results in half perceived loudness.
+// VolumeScale = -1000 / log(0.5)
+constexpr float VolumeScale = 3321.9281f;
+
+// Min and max volume range, in millibel.
+// -100 dB (muted) to 0 dB (max. loudness).
+constexpr float MillibelMin = -10000.0f;
+constexpr float MillibelMax = 0.0f;
+
+// Stereo separation factor for left/right speaker panning. Lower values increase separation, moving sounds further left/right, while higher values will pull sounds more towards the middle, reducing separation.
+// Current value is tuned to have ~2:1 mix for sounds that happen on the edge of a 640x480 screen.

```

Comment:
Break up extremely long lines, if they do not fit on screen they are hard to read...
```suggestion
/**
 * Stereo separation factor for left/right speaker panning. Lower values increase separation, moving
 * sounds further left/right, while higher values will pull sounds more towards the middle, reducing separation.
 * Current value is tuned to have ~2:1 mix for sounds that happen on the edge of a 640x480 screen.
 */
```